### PR TITLE
Change headers and add cost calculation

### PR DESCRIPTION
**Pull request by knikolla**

### CODE REVIEW

File: src/coldfront_plugin_cloud/management/commands/calculate_storage_gb_hours.py [hunk 1]
```diff
@@ -24,28 +25,40 @@ class InvoiceRow:
...
+    Cost: Decimal = 0
+
 
     @classmethod
     def get_headers(cls):
-        """Returns all fields for display, with space instead of underscore."""
+        """Returns all headers for display."""
         return [
-            field.name.replace("_", " ") for field in dataclasses.fields(cls)
+            "Interval",
```
Comment (joachimweyl) [inline src/coldfront_plugin_cloud/management/commands/calculate_storage_gb_hours.py:38]: Invoice Month

Comment (knikolla) [reply src/coldfront_plugin_cloud/management/commands/calculate_storage_gb_hours.py:38]: The tool accepts arbitrary start and end dates that don't have to conform to an exact month, therefore with the current logic of the code the header Invoice Month would be incorrect.

Comment (joachimweyl) [reply src/coldfront_plugin_cloud/management/commands/calculate_storage_gb_hours.py:38]: That is the name of the Header as it houses only a specific month (eg. 2023-09). The value of the field in the code might hold something else during the processing of that output but the header needs to match the header of the CSV we are sending to MGHPCC.

Comment (knikolla) [reply src/coldfront_plugin_cloud/management/commands/calculate_storage_gb_hours.py:38]: Yes, right now I am changing the header and entire column manually before sending it to you. I will change the tool to output the header that you expect (Invoice Month) in a follow up patch as that requires adding new parameters to the execution of the tool.

**Review by joachimweyl — APPROVED** [review]

**Comment by knikolla** [comment]
Job `test-py39-functional-microshift` was slightly flaky. Re-running it fixed the issue.

**Pull request merged**